src/reduced_basis/rb_eim_construction.C

```diff
@@ -217,6 +217,12 @@ Real RBEIMConstruction::train_reduced_basis(const bool resize_rb_eval_data)
 Number RBEIMConstruction::evaluate_mesh_function(unsigned int var_number,
                                                  Point p)
 {
+  // Set default values to be an empty vector so that we can use it
+  // below
```

Comment:
Formatting of this comment is a little weird, might want to move "here" to the previous line